srv/modules/runners/validate.py

```diff
@@ -1141,9 +1174,22 @@ def prep(**kwargs):
 
 def setup(**kwargs):
     """
-    Check that initial files prior to any stage are correct
+    Check that initial files prior to any stage are correct.  These
+    validations are intended for fresh deployments.  Skip automatic
+    checks on working clusters.
     """
     printer = get_printer(**kwargs)
+    if ('bypass' in kwargs and kwargs['bypass'] and
+        __salt__['cephprocesses.mon']()):
+        # Disable all Salt lookups
+        valid = Validate("setup", search_pillar=False, search_grains=False,
+                         skip_init=True, printer=printer)
+        valid.skip('deepsea_minions')
+        valid.skip('master_minion')
+        valid.skip('ceph_version')
```

Comment:
If we disable those after the initial deployment, we have no way to identify if an update brought in a ceph version that we don't support. For salt_version we have the RPM that is supposed to conflict with any unsupported version.